packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -408,6 +423,10 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
                 aria-autocomplete={this._getAriaAutoCompleteValue()}
                 role="combobox"
                 readOnly={disabled || !allowFreeform}
+                aria-posinset={this._sizePosCache.positionInSet(
+                  currentPendingValueValidIndex > 0 ? currentPendingValueValidIndex : selectedIndices[0]
```

Comment:
Maybe ` currentPendingValueValidIndex >= 0` instead? Looks like `0` is a valid value if I'm reading existing code correctly.

Reply:
Fixed.